### ISSUE TITLE
'Use same address for billing' can be checked/unchecked in the editor, but its status isn't reflected in the frontend

### ISSUE THREAD

**Issue by Aljullu** [woothemes/woocommerce#42623]
## Describe the bug

Changing the status of the _Use same address for billing_ checkbox in the editor doesn't have any impact in the frontend.

## To reproduce

1. Add the Checkout block to a post or page.
2. In the editor, notice you can check/uncheck the _Use same address for billing_ checkbox. That shows the billing section or not. Leave it unchecked.
3. Visit the Checkout block in the frontend.
4. Notice the _Use same address for billing_ checkbox is checked by default and the billing sections isn't visible.

To me, that's a bit confusing: why can the user check/uncheck the _Use same address for billing_ checkbox in the editor if that won't have any impact in the frontend?

## Expected behavior

If I uncheck the _Use same address for billing_ checkbox in the editor, I would expect it to be unchecked by default in the frontend.

## Screenshots

Editor | Frontend
--- | ---
![imatge](https://user-images.githubusercontent.com/3616980/174614877-50986104-6ecd-4346-b22e-9db175b4bdac.png) | ![imatge](https://user-images.githubusercontent.com/3616980/174614940-19507295-3afa-4965-831c-a5e88b69dbe9.png)

**Comment by Aljullu** [comment]
Related: https://github.com/woocommerce/woocommerce-blocks/pull/4457.

**Comment by ralucaStan** [comment]
Related https://wordpress.org/support/topic/uncheck-billing-address-same-as-shipping-address-by-default/

**Comment by LucaRed** [comment]
The easiest solution (from a UX standpoint) would be to persist the state of the checkbox in the block editor as the default option for users (i.e. if I uncheck it in the block editor, users will see the Billing Address by default).

Alternatively, a toggle could be added in the settings panel.

**Comment by kmanijak** [comment]
Related: https://wordpress.org/support/topic/same-as-shipping-address-by-default/

A user would like to be able to uncheck the billing address “Same as Shipping Address” by default. While it's possible to check/uncheck in the Editor, it doesn't reflect in the frontend.

**Comment by jarman1** [comment]
Hi All. 

Any updates on this - It prevents B2B mechants using the checkout block as it causes friction at checkout and potential returns due to wrong shipping addresses being used:

A neat solution would be a toggle to leave the shipping address fully open and the purchaser just clicks use the billing address for shipping - this closes the shipping address down. Essentially the reverse of what is the default currently.

**Comment by mikejolley** [comment]
I think the majority of the UI in the editor should be `<disabled>` to avoid setting a precedent that form elements in the preview affect the frontend display.

It could be an inspector toggle, or just a filter, to change the default behaviour.

**Comment by jarman1** [comment]
Any Updates?

**Comment by mikejolley** [comment]
To summarise the issue for anyone coming in:

- The toggle in the editor does not persist anything, its just used to preview what "billing" will look like
- The billing block needs to be editable because of the form step heading and description text which can be customised

We'll get design feedback for this before tackling it.

**Comment by jarman1** [comment]
Thansks - Appreciate update. Really critical for UX & conversion optimisation for B2B where shipping address is often different to billing.

Thanks again

**Comment by ralucaStan** [comment]
@pierorocca for awareness

**Comment by jarman1** [comment]
Has this been resolved yet?

**Comment by ralucaStan** [comment]
No, we haven't worked on this and I don't think we'll have the capacity to redesign and implement a new approach. So I am closing the issue for now.
